lazer/contracts/evm/script/PythLazerDeploy.s.sol

```diff
@@ -168,8 +169,25 @@ contract PythLazerDeployScript is Script {
         return addr;
     }
 
+    function getProxyAddress(bytes11 seed) public view returns (address addr) {
+        (, bytes32 guardedSalt) = generateSalt(seed);
+        address proxyAddr = createX.computeCreate3Address({salt: guardedSalt});
+        return proxyAddr;
+    }
+
     function run() public {
         address impl = deployImplementation("lazer:impl");
         deployProxy("lazer:proxy", impl);
     }
+
+    function migrate() public {
+        // Deploys new version and updates proxy to use new address
+        address proxyAddress =  getProxyAddress("lazer:proxy");
```

Comment:
This should deploy to a different address than the previous version since the code itself is used as part of the hash to produce the address.